index.html

```diff
@@ -34,12 +34,236 @@ <h3>I am a software developer</h3>
+                                    <p class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <ul class="tags">
+                                        <li class="tags1"><h6>html</h6></li>                                            
+                                        <li class="tags2"><h6>bootstrap</h6></li>
+                                        <li class="tags3"><h6>Ruby</h6></li>
+                                    </ul>
+                                </section>
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <ul class="tags">
+                                        <li class="tags1"><h6>html</h6></li>                                            
+                                        <li class="tags2"><h6>bootstrap</h6></li>
+                                        <li class="tags3"><h6>Ruby</h6></li>
+                                    </ul>
+                                </section>
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+            </section>
+            <section class="about">
+                <p class="primary">About Me</p>
+                <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <section class="img-about"><img src="images/illustration-about-me-mobile.png" alt=""></section>
+                <section class="button">
+                    <p class="get">Get My Resume</p>
```

Comment:
- [ ] As the last code reviewer said: Please 🙏🏽 use the `button` tag or the `a` tag for all your buttons. This is the best HTML practice.